specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -1368,495 +1368,2776 @@
+        },
+        "ipRangeFilter": {
+          "description": "Cosmos DB Firewall Support: This value specifies the set of IP addresses or IP address ranges in CIDR form to be included as the allowed list of client IPs for a given database account. IP addresses/ranges must be comma separated and must not contain any spaces.",
+          "$ref": "#/definitions/IPRangeFilter"
+        },
+        "isVirtualNetworkFilterEnabled": {
+          "description": "Flag to indicate whether to enable/disable Virtual Network ACL rules.",
+          "type": "boolean"
+        },
+        "enableAutomaticFailover": {
+          "description": "Enables automatic failover of the write region in the rare event that the region is unavailable due to an outage. Automatic failover will result in a new write region for the account and is chosen based on the failover priorities configured for the account.",
+          "type": "boolean"
+        },
+        "consistencyPolicy": {
+          "description": "The consistency policy for the Cosmos DB database account.",
+          "$ref": "#/definitions/ConsistencyPolicy"
+        },
+        "capabilities": {
+          "type": "array",
+          "description": "List of Cosmos DB capabilities for the account",
+          "items": {
+            "$ref": "#/definitions/Capability"
+          }
+        },
+        "writeLocations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains the write location for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        },
+        "readLocations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains of the read locations enabled for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        },
+        "failoverPolicies": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains the regions ordered by their failover priorities.",
+          "items": {
+            "$ref": "#/definitions/FailoverPolicy"
+          }
+        },
+        "virtualNetworkRules": {
+          "type": "array",
+          "description": "List of Virtual Network ACL rules configured for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/VirtualNetworkRule"
+          }
+        },
+        "enableMultipleWriteLocations": {
+          "description": "Enables the account to write in multiple locations",
+          "type": "boolean"
+        }
+      }
+    },
+    "DatabaseAccountPatchProperties": {
+      "description": "Properties to update Azure Cosmos DB database accounts.",
+      "type": "object",
+      "properties": {
+        "capabilities": {
+          "type": "array",
+          "description": "List of Cosmos DB capabilities for the account",
+          "ite
```

Comment:
Getting a unmarshalling error here preventing all reads:
```
* azurerm_cosmos_sql_container.test:Error making get request for Cosmos SQL Container seven-day-container (Account kt-cosmos-sql,
Database seven-day-db):documentdb.DatabaseAccountsClient#GetSQLContainer:Failure responding to request:StatusCode=200 -- Original Error:Error occurred unmarshalling JSON - Error = 'json:cannot unmarshal object into Go struct field IndexingPolicy.excludedPaths of type string' JSON = '{  
   "properties":{  
      "id":"seven-day-container",
      "indexingPolicy":{  
         "indexingMode":"consistent",
         "automatic":true,
         "includedPaths":[  
            {  
               "path":"/*",
               "indexes":[  
                  {  
                     "kind":"Range",
                     "dataType":"Number",
                     "precision":-1
                  },
                  {  
                     "kind":"Range",
                     "dataType":"String",
                     "precision":-1
                  }
               ]
            }
         ],
         "excludedPaths":[  
            {  
               "path":"/\"_etag\"/?"
            }
         ]
      },
      "conflictResolutionPolicy":{  
         "mode":"LastWriterWins",
         "conflictResolutionPath":"/_ts",
         "conflictResolutionProcedure":""
      },
      "_rid":"639ZAJUnyLc=",
      "_ts":1555048744,
      "_self":"dbs/639ZAA==/colls/639ZAJUnyLc=/",
      "_etag":"\"0000ec30-0000-0a00-0000-5cb029280000\"",
      "_docs":"docs/",
      "_sprocs":"sprocs/",
      "_triggers":"triggers/",
      "_udfs":"udfs/",
      "_conflicts":"conflicts/"
   }
}'
```

Reply:
excludedPaths is fixed in the new commit. it should be array of object.